data-service/server/src/resolvers.js

```diff
@@ -24,6 +26,11 @@ const queries = {
     }`;
   },
 };
+const userHasRole = (user, role) => {
+  if (!user.roles || !user.roles.length) return;
```

Comment:
If we have babel setup with null coarsening operator, can do `if (!user?.roles?.length) ...`

Reply:
4792599